backend/src/main/java/com/cruru/applyform/service/ApplyFormService.java

```diff
@@ -91,14 +100,57 @@ private void validatePersonalDataCollection(ApplyFormSubmitRequest request) {
 
     private void saveAnswerReplies(AnswerCreateRequest answerCreateRequest, Applicant applicant) {
         for (String reply : answerCreateRequest.replies()) {
-            Question question = getQuestionById(answerCreateRequest.questionId());
+            Question question = questionRepository.findById(answerCreateRequest.questionId())
+                    .orElseThrow(QuestionNotFoundException::new);
             Answer answer = new Answer(reply, question, applicant);
             answerRepository.save(answer);
         }
     }
 
-    private Question getQuestionById(long questionId) {
-        return questionRepository.findById(questionId)
-                .orElseThrow(QuestionNotFoundException::new);
+    public ApplyFormResponse read(long applyFormId) {
+        ApplyForm applyForm = applyFormRepository.findById(applyFormId)
+                .orElseThrow(ApplyFormNotFoundException::new);
+
+        List<QuestionResponse> responses = questionRepository.findAllByApplyFormId(applyFormId)
```

Comment:
```suggestion
        List<QuestionResponse> questionResponses = questionRepository.findAllByApplyFormId(applyFormId)
```
해당 메서드내에서 ApplyFormResponse, QuestionResponse를 모두 다루고 있어, 변수에 이름을 명시하는 것이 가독성이 좋다고 생각하는데요. 초코칩은 어떻게 생각하시나요?

Reply:
바꿔보니까 가독성이 더 좋은 것 같네요. 반영했습니다.